egs2/voxforge/asr1/run.sh

```diff
@@ -10,11 +10,18 @@ train_set="tr_${lang}"
 dev_set="dt_${lang}"
 eval_sets="et_${lang}"
 
+lm_config=conf/lm_rnn/train.yaml
+asr_config=conf/asr_rnn/train.yaml
+decode_config=conf/asr_rnn/decode.yaml
+
 
 ./asr.sh \
     --local_data_opts "--lang ${lang}" \
     --token_type char \
-    --lm_config conf/lm_train.yaml \
+    --feats_type fbank_pitch \
+    --lm_config "${lm_config}" \
```

Comment:
why did you add lm and decode configs?

Reply:
just for example

l  ll add an option to skip lm training and decode without lm.